README.md

```diff
@@ -1,19 +1,12 @@
 # cosign
+
 Sigstore cosign repository for Home Assistant
 
 
 ## Binary
 
-Binary files for our Alpine Linux based Docker images.
-Prebuild for speed-up the build process with UPX.
+Binary files for our Alpine Linux based Docker images for all Home Assistant
+supported architectures. Prebuild using GitHub Actions to speed-up the build
+process of Docker containers like the builder.
 
 Source: https://github.com/sigstore/cosign/releases
-
-Commands:
-
-- CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -trimpath -o cosign_amd64 ./cmd/cosign
-- CGO_ENABLED=0 GOOS=linux GOARCH=386 go build -trimpath -o cosign_i386 ./cmd/cosign
-- CGO_ENABLED=0 GOOS=linux GOARCH=arm GOARM=7 go build -trimpath -o cosign_armv7 ./cmd/cosign
-- CGO_ENABLED=0 GOOS=linux GOARCH=arm GOARM=6 go build -trimpath -o cosign_armhf ./cmd/cosign
-- GO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -trimpath -o cosign_aarch64 ./cmd/cosign
-- upx -9 cosign_amd64 cosign_i386 cosign_armv7 cosign_armhf cosign_aarch64
```

Comment:
Is it intentional that UPX packing is not done anymore?

Reply:
We should have it to make our image small

Reply:
I noticed it after submitting the review. @agners can you add a step with that? Also I think you'll have to add another step to install it from apt.

Reply:
I've opted to remove it since the layers in our container images getting compressed anyways, afaik? Also, uncompromising the binary on-the-fly adds overhead on each invocation, especially when running in emulated mode (and we do multiple calls to cosign in the builder).

Lastly, it would add complexity to the GH action.

Instead, I've added this, which saves about half of the savings UPX did: `-ldflags="-s -w"`.

Reply:
Some direct discussion: Disk space savings are still bigger with UPX. But less with the ldflags. We decided to leave out UPX, at least for now.